client/app/src/accounts/account.controller.js

```diff
@@ -38,6 +38,25 @@
         return username
       }
     }])
+    .filter('formattedDate', ['storageService', '$filter', function (storageService, dateFilter) {
```

Comment:
Unit tests for this formattedDate filter would be a good idea

Reply:
I'm new in Angular and JS in general, but I'll learn how to create that test and then I'll submit it.

Reply:
Thanks for your effort!